### ISSUE TITLE
Feature Request: add a new widget type which can display an image. 

### ISSUE THREAD

**Issue by Yrani6** [chriskiehl/Gooey#840]
I want to embed an image in the GUI page. How do I do that?

**Comment by chriskiehl** [comment]
If you mean embed one in the terminal output? it's not currently supported. 
If you mean change one of the images used throughout gooey itself: checkout the image section of the readme.

**Comment by Yrani6** [comment]
> 如果您的意思是在终端输出中嵌入一个？目前不支持。 如果您的意思是更改整个 gooey 本身使用的图像之一：查看自述文件的图像部分。

I need a picture presentation component

**Comment by chriskiehl** [comment]
Gooey wraps command line programs. There's no built in presentation component. 

You could work around it by popping a second window, but that's about as good as it gets. 


```python
from gooey import Gooey, GooeyParser, options
from tkinter import *

@Gooey(auto_start=True)
def main():
    parser = GooeyParser(description='Just display the console')
    parser.add_argument('image', widget='FileChooser', gooey_options=options.FileChooser(wildcard='*.png'))
    args = parser.parse_args()
    print("Check out this image! Close the window when you're done.")

    ws = Tk()
    img = PhotoImage(file=args.image)
    Label(ws, image=img).pack()
    ws.mainloop()
    print("All done!")


if __name__ == '__main__':
    main()
```

![image](https://user-images.githubusercontent.com/1408720/188249313-3b6eb106-9ebf-4677-a10a-61fbf54736b9.png)

**Comment by Yrani6** [comment]
> Gooey 封装了命令行程序。没有内置的演示组件。
> 
> 您可以通过弹出第二个窗口来解决它，但这已经差不多了。
> 
> ```python
> from gooey import Gooey, GooeyParser, options
> from tkinter import *
> 
> @Gooey(auto_start=True)
> def main():
>     parser = GooeyParser(description='Just display the console')
>     parser.add_argument('image', widget='FileChooser', gooey_options=options.FileChooser(wildcard='*.png'))
>     args = parser.parse_args()
>     print("Check out this image! Close the window when you're done.")
> 
>     ws = Tk()
>     img = PhotoImage(file=args.image)
>     Label(ws, image=img).pack()
>     ws.mainloop()
>     print("All done!")
> 
> 
> if __name__ == '__main__':
>     main()
> ```
> 
> ![图片](https://user-images.githubusercontent.com/1408720/188249313-3b6eb106-9ebf-4677-a10a-61fbf54736b9.png)

![image](https://user-images.githubusercontent.com/64725124/188249847-74bed8a8-0a2f-4166-a0a3-2c9bb8b28751.png)
Whether to show a picture here？

**Comment by Yrani6** [comment]
> > Gooey 组装了一套程序。没有组装的演示组件。
> > 你可以通过弹出一个窗口来解决它，这已经差不多了。
> > ```python
> > from gooey import Gooey, GooeyParser, options
> > from tkinter import *
> > 
> > @Gooey(auto_start=True)
> > def main():
> >     parser = GooeyParser(description='Just display the console')
> >     parser.add_argument('image', widget='FileChooser', gooey_options=options.FileChooser(wildcard='*.png'))
> >     args = parser.parse_args()
> >     print("Check out this image! Close the window when you're done.")
> > 
> >     ws = Tk()
> >     img = PhotoImage(file=args.image)
> >     Label(ws, image=img).pack()
> >     ws.mainloop()
> >     print("All done!")
> > 
> > 
> > if __name__ == '__main__':
> >     main()
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ![图片](https://user-images.githubusercontent.com/1408720/188249313-3b6eb106-9ebf-4677-a10a-61fbf54736b9.png)
> 
> ![图片](https://user-images.githubusercontent.com/64725124/188249847-74bed8a8-0a2f-4166-a0a3-2c9bb8b28751.png) 此处是否显示图片？

Could future maintenance include an image component like the one above?

**Comment by chriskiehl** [comment]
Yep. I understand now. That's a reasonable request to me and easy to add implement. 

I've reopened this and renamed it to track the feature request .
